code/game/objects/items/plushes.dm

```diff
@@ -546,6 +546,15 @@
 	gender = FEMALE
 	squeak_override = list('sound/voice/moth/scream_moth.ogg'=1)
 
+/obj/item/toy/plush/georgefloyd
+	name = "George 'Venom' Floyd"
+	desc = "Gone too soon. RIP George Floyd"
+	icon_state = "george_floyd"
+	item_state = "george_floyd"
+	attack_verb = list("took breath away from")
```

Comment:
this shows up weirdly ingame, might be best to remove its attack verb or change it to default value

Other than that, I like the changes.

Reply:
`attack_verb = list("took breath away from")`

Reply:
Doesn’t it show as LastCrusader took breath away from Dapper?
Your right sounds wrong maybe took away breath from dapper?

Reply:
If you really want to keep the taking breath aspect of this, which I find pretty distasteful tbh

do something like, `chokes` or `smothers` or something
rather than something long ygm

Reply:
Fine chokes it is
Choke me daddy

the whole situation was distasteful
Which is why we must never forget he was choked